dbsa/presto.py

```diff
@@ -166,3 +166,33 @@ def get_create_current_partition_view(self, suffix='_latest', condition='', igno
             select=self.get_select_current_partition(condition=condition, ignored_partitions=ignored_partitions, params=params, transforms=transforms),
             suffix=suffix,
         )
+
+    def get_incremental_update_select(self, update_select, row_identifier=None, filter_fn=None, condition='', ignored_partitions=None, params=None, transforms=None):
```

Comment:
Row Identifier should be mandatory, and we should expect lists. 

E.g.: `["id"]`

Reply:
Why? I can imagine scenarios where you only extend.

Reply:
No filter just extend the previous data

Reply:
For example
```sql
      WITH incremental_update AS (                
        SELECT 
          object_id, field, add_date
        FROM data.update_table
        WHERE ds = '{{ ds }}'
      ),
      recent_data AS (          
        SELECT
          object_id, field, add_date
        FROM data.base_table
        WHERE ds = '{{ yesterday_ds }}'
       )
       SELECT *
       FROM incremental_update
       UNION ALL
       SELECT * 
       FROM recent_data
```